docker/glibc/README.rst

```diff
@@ -0,0 +1,79 @@
+centos-6-no-vsyscall
+====================
+
+*Summary*: Because of
+https://mail.python.org/pipermail/wheel-builders/2016-December/000239.html,
+this a CentOS 6.10 Docker image that rebuilds ``glibc`` without
+*vsyscall* is necessary to reliably run ``manylinux2`` on 64-bit
```

Comment:
manylinux2 -> manylinux2010 throughout this document

Reply:
Done, thanks @JonasVautherin